model/src/main/kotlin/OrtResult.kt

```diff
@@ -76,18 +79,27 @@ data class OrtResult(
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val data: CustomData = mutableMapOf()
 
-    private val excludedProjects: Set<Identifier> by lazy {
-        getProjects()
-            .filter { project ->
+    private data class ProjectEntry(val project: Project, val isExcluded: Boolean)
+
+    private val projects: Map<Identifier, ProjectEntry> by lazy {
```

Comment:
A KDoc would be helpful, explainng the benefit of this property.

Reply:
I just tried to write a comment but all ideas coming to my mind felt wrong as those seemed to describe the obvious. Can you give me a hint what you had in mind?

Reply:
About the same you wrote in the commit message. Explain that this is required to improve the performance of looking up projects and if they are excluded because it is down very often in the evaluator and reporter.

But now that I think about it again, this explanation should actually go to the public API function `getProject()` to explain to the user why this function should be used instead of looking up the projects in the analyzer result.

Reply:
As discussed not mandatory fixing.